packages/eslint-config-sentry-app/index.js

```diff
@@ -142,6 +150,22 @@ module.exports = {
       },
     ],
 
+    /**
+     * Warnings for imports still in use but will be removed
+     */
+    'no-restricted-imports': [
+      'warn',
```

Comment:
moved enzyme -> RTL rule to warning as it does not make sense to have the rule and add `// eslint-disable-next-line no-restricted-imports` in all not yet converted tests.. or should we do that? 🤷‍♀️

Reply:
> RTL rule to warning as it does not make sense to have the rule and add // eslint-disable-next-line no-restricted-imports in all not yet converted tests.. or should we do that

A warning is fine for now, as we get closer to complete, we can upgrade to an error. We are still early in the process though, and you'll have to churn ~1000 files right now.

Reply:
You could make it an error in the [strict](https://github.com/getsentry/eslint-config-sentry/blob/master/packages/eslint-config-sentry-app/strict.js) config -- it's only used in pre commit hooks.